src/helpers.c

```diff
@@ -521,27 +521,26 @@ void free_file_data(void *data)
 	free(file);
 }
 
-void swupd_deinit(int lock_fd, struct list **subs)
+void swupd_deinit(void)
 {
 	terminate_signature();
 	swupd_curl_deinit();
-	if (subs) {
-		free_subscriptions(subs);
-	}
 	free_globals();
-	v_lockfile(lock_fd);
+	v_lockfile();
 	dump_file_descriptor_leaks();
 }
 
 /* this function is intended to encapsulate the basic swupd
 * initializations for the majority of commands, that is:
 * 	- Make sure root is the user running the code
-*	- Initialize log facility
-*	- Get the lock
+*       - Initialize globals
 *	- initialize mounted directories
+*       - Create necessary directories
+*	- Get the lock
 *	- Initialize curl
+*       - Initialize signature checking
```

Comment:
Are those items in the comments with a different alignment on purpose? Are they like sub-bullets?